src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.cs

```diff
@@ -0,0 +1,1418 @@
+        Target: IMethodBindingExpression: void Program.M1(System.Object o) (OperationKind.MethodBindingExpression, Type: null, IsInvalid) (Syntax: 'M1')
+            Instance Receiver: IInstanceReferenceExpression (OperationKind.InstanceReferenceExpression, Type: Program, IsInvalid) (Syntax: 'M1')
+";
+            var expectedDiagnostics = new DiagnosticDescription[] {
+                // CS0123: No overload for 'Program.M1(object)' matches delegate 'Action<int>'
+                //         /*<bind>*/Action<int> a = M1;/*</bind>*/
+                Diagnostic(ErrorCode.ERR_MethDelegateMismatch, "M1").WithArguments("Program.M1(object)", "System.Action<int>").WithLocation(7, 35)
+            };
+
+            VerifyOperationTreeAndDiagnosticsForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void DelegateCreationExpression_ImplicitMethodBinding_MultipleCandidates()
+        {
+            string source = @"
+using System;
+class Program
+{
+    void Main()
+    {
+        /*<bind>*/Action<int> a = M1;/*</bind>*/
+    }
+    void M1(object o) { }
+    void M1(int i) { }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Action<int> a = M1;')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'Action<int> a = M1;')
+    Variables: Local_1: System.Action<System.Int32> a
+    Initializer: IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Action<System.Int32>) (Syntax: 'M1')
+        Target: IMethodBindingExpression: void Program.M1(System.Int32 i) (OperationKind.MethodBindingExpression, Type: null) (Syntax: 'M1')
+            Instance Receiver: IInstanceReferenceExpression (OperationKind.InstanceReferenceExpression, Type: Program) (Syntax: 'M1')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void DelegateCreationExpression_ExplicitDelegateConstructorImplicitMethodBinding_MultipleCandidates()
+        {
+            string source = @"
+using System;
+class Program
+{
+    void Main()
+    {
+        Action<string> a = /*<bind>*/new Action<string>(M1)/*</bind>*/;
+    }
+
+    void M1(object o) { }
+
+    void M1(string s) { }
+}
+";
+            string expectedOperationTree = @"
+IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Action<System.String>) (Syntax: 'new Action<string>(M1)')
+  Target: IOperation:  (OperationKind.None) (Syntax: 'M1')
```

Comment:
> Target: IOperation:  (OperationKind.None) (Syntax: 'M1') [](start = 1, length = 57)

This is unexpected. It looks like there are no errors, why are we not getting the method reference?

Reply:
I'm not sure what you mean? We are getting a method reference.

---
In reply to: [140107408](https://github.com/dotnet/roslyn/pull/22056#discussion_r140107408) [](ancestors = 140107408)

Reply:
> I'm not sure what you mean? We are getting a method reference.

We didn't in iteration 22.

---
In reply to: [141477343](https://github.com/dotnet/roslyn/pull/22056#discussion_r141477343) [](ancestors = 141477343,140107408)